TestProd/src/com/recapcha/servlet/LoginServlet.java

```diff
@@ -0,0 +1,55 @@
+package com.recapcha.servlet;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LoginServlet extends HttpServlet
+{
+	
+	
+	private static final long serialVersionUID = 1L;
+	
+	@Override
+	protected void doPost(HttpServletRequest request,HttpServletResponse response) throws ServletException, IOException 
+	{
+
+					String user = request.getParameter("user");
+					String pwd = request.getParameter("pwd");
```

Comment:
Not a good practice to take passwords in variable. Directly use it wherever required.